tests/core/shader.test.js

```diff
@@ -152,6 +152,114 @@ suite('shader', function () {
                                            instance.attributes['src']));
     });
 
+    // Skip since this fails Travis CI Firefox run, even though it works on Chrome, and with local Firefox.
+    test.skip('iOS HLS video uses appropriate shader', function (done) {
+      var shader = this.shader;
+      var el = this.el;
+      var initSpy = this.sinon.spy(shader.prototype, 'init');
+      var updateSpy = this.sinon.spy(shader.prototype, 'update');
+      assert.notOk(initSpy.called);
+      assert.notOk(updateSpy.called);
+
+      // Mock iOS.  NOTE: this doesn't work... el.sceneEl.isIOS = true;
+      var realIsIOS = AFRAME.utils.device.isIOS;
+      AFRAME.utils.device.isIOS = function () { return true; };
```

Comment:
`this.sinon.stub(AFRAME.utils.device.isIOS, () => true)` might work, if we use sinon, the teardown will automatically restore it.